integration-tests/pkg/types/endpoint.go

```diff
@@ -62,5 +62,9 @@ func endpointComparison(endpoint1 EndpointInfo, endpoint2 EndpointInfo) bool {
 		return process1.ProcessArgs < process2.ProcessArgs
 	}
 
+	if endpoint1.IsActive() == endpoint2.IsActive() {
```

Comment:
This stands a bit out of the pattern used in this function -- all of them are compared for not equality. Maybe to reduce the surprise level, compare for inequality here as well?